testsuite/features/secondary/srv_cobbler_distro.feature

```diff
@@ -6,18 +6,22 @@ Feature: Cobbler and distribution autoinstallation
 
   Scenario: Log in as testing user
     Given I am authorized as "testing" with password "testing"
+    And I am logged via the Cobbler API as user "testing" with password "testing"
```

Comment:
```suggestion
    And I am logged in via the Cobbler API as user "testing" with password "testing"
```
Still one missing :wink:

Reply:
Forgot to add this file to the commit, thanks for noticing!